### PR TITLE
Fixed different font-size for labels in config screen

### LINKED ISSUE #1788: Different font-size for form labels with Material theme
### Issue details
On config form, labels aren't displayed with same font-size (ex with first tab):
- "Theme": 12px
- "Items per page": 15px

By the way, Theme's label should be placed before ".select-wrapper" in HMTL code to avoid a blank space due to the margin-top of .select-wrapper".

Labels from RSS and Password tabs have same problem.

### Environment
2.0.0-beta.2 via http://v2.wallabag.org/about
FF 45.0 on MacOS X

### PR DESCRIPTION
**Pull request by nicosomb**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Documentation | no
| Translation   | no
| Fixed tickets | #1788 
| License       | MIT

Fix #1788

### CODE REVIEW

File: src/Wallabag/CoreBundle/Resources/views/themes/material/Config/index.html.twig [hunk 1]
```diff
@@ -196,6 +196,7 @@
                     </div>
 
                     <div id="set5" class="col s12">
+                        {% if app.user.config.taggingRules is not empty %}
```
Comment (nicosomb) [inline src/Wallabag/CoreBundle/Resources/views/themes/material/Config/index.html.twig]: this test is here to remove useless lines when the user didn't create any tagging rule.

**Comment by j0k3r** [comment]
Screenshot before/after?

**Comment by nicosomb** [comment]
Before 
before

After
after

About tagging rules, you can see the useless space here: 
tagging

**Comment by j0k3r** [comment]
:+1:

**Pull request merged**